src/test/software/amazon/event/ruler/ComparableNumberTest.java

```diff
@@ -139,4 +139,46 @@ public void epocTimestampRangeTest() throws Exception {
         }
         assertTrue(m.isEmpty());
     }
+
+    static String generateNew(String number) {
+        double f = FastDoubleParser.parseDouble(number);
+        return ComparableNumber.generate(f);
+    }
+
+    static String generateOriginal(String number) {
+        double f = Double.parseDouble(number);
+        return ComparableNumber.generate(f);
+    }
```

Comment:
This can be now private. Probably should be after the test as well (rest of the code follows a similar pattern).